packages/name-resolution/src/sid/index.ts

```
@@ -1,72 +1,114 @@
+
+    const tld = getTLD(name);
+    switch (tld) {
+      case "sol":
+        const solAddress = await this.solanaNameResolver.getAddress({
+          name,
+        });
+        return solAddress;
+      default:
+        const address = await this.web3NameResolver.getAddress(name, {
+          timeout: this.timeout,
+        });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)